src/replication_test.c

```diff
@@ -75,22 +75,24 @@ init_mdlist(char *vol_name)
 
 	mdlist_fd = open(mdpath, O_CREAT|O_RDWR, 0666);
 	if (mdlist_fd < 0) {
-		REPLICA_ERRLOG("Failed to open metadata file %s err(%d)\n",
+		REPLICA_ERRLOG("Failed to open"
```

Comment:
this seems to be unnecessary change. This is not part of cstyle errors of this file replication_test.c

Reply:
It was showing me error on this line (when I ran it on my pc) @vishnuitta

Reply:
can I know the errors that you saw?

Reply:
Cstyle character>80 warning

Reply:
Cstyle character>80 warning @vishnuitta